File: app/src/main/java/com/ws/design/coco_ecommerce_ui_kit/SplashActivity.java
package com.ws.design.coco_ecommerce_ui_kit;


import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.os.Handler;

import com.example.wolfsoft2.coco_ecommerce_ui_kit.R;

public class SplashActivity extends AppCompatActivity {

    private final int SPLASH_DISPLAY_LENGTH = 1000;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_screen);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent = new Intent(SplashActivity.this, StartActivity.class);
                SplashActivity.this.startActivity(intent);
                SplashActivity.this.finish();
            }
        }, SPLASH_DISPLAY_LENGTH);
    }
}

File: app/src/main/java/com/richkart/android/SplashActivity.kt
package com.richkart.android

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.support.v7.app.AppCompatActivity
import com.crashlytics.android.Crashlytics
import io.fabric.sdk.android.Fabric

class SplashActivity : AppCompatActivity(){

    private val SPLASH_DISPLAY_LENGTH = 1000

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_screen)
        Fabric.with(this, Crashlytics())


        Handler().postDelayed({
            val intent = Intent(this@SplashActivity, StartActivity::class.java)
            this@SplashActivity.startActivity(intent)
            this@SplashActivity.finish()
        },SPLASH_DISPLAY_LENGTH.toLong())


    }
}